<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourWidget</class>
 <widget class="QWidget" name="ContourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Height band:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bandSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="elevCheckBox">
     <property name="text">
      <string>Set elevation attribute:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="elevLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>ELEV</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="treeDCheckBox">
     <property name="text">
      <string>Make 3D vertexes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ignoreNodataCheckBox">
     <property name="text">
      <string>Ignore nodata</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="skipCheckBox">
       <property name="text">
        <string>Skip pixel values:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="skipSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="regularRadioButton">
       <property name="text">
        <string>Regular interval: </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="regularIntervalSpinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix">
        <string> m</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="customRadioButton">
     <property name="text">
      <string>Custom interval levels:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="levelsListView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addLevButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delLevButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearLevButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>clr</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>86</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>elevCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>elevLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>skipCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>skipSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regularRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>regularIntervalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>addLevButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>delLevButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>clearLevButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>levelsListView</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
